src/metabase/models/action.clj

```diff
@@ -250,7 +250,7 @@
   "Hydrates action from DashboardCard."
   [dashcards]
   (let [actions-by-id (when-let [action-ids (seq (keep :action_id dashcards))]
-                        (m/index-by :id (select-actions (map :card dashcards) :id [:in action-ids])))]
```

Comment:
This might have worked if the `:card` of an implicit action dashcard contained the card of the action. But it doesn't. The `card_id` for implicit actions is `null` and the hydrated `card` is `{query_average_duration: null}`.

Reply:
should this comment be reflected in the code?

Reply:
I've asked the question on [slack](https://metaboat.slack.com/archives/CKZEMT1MJ/p1676623946326239). I'll make a follow up PR